### PR TITLE
CTAA-1364 - Screen for informing user that he can delete his exposure…

### PR DESCRIPTION
**Pull request by mihbt**
## Changes

Screen to inform user how he can delete his exposure keys.

## Solved issues

- [Issue #1398](https://tasks.pxp-x.com/browse/CTAA-1398)

### CODE REVIEW

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import at.roteskreuz.stopcorona.screens.base.CoronaPortraitBaseActivity
+import at.roteskreuz.stopcorona.skeleton.core.screens.base.activity.startFragmentActivity
+import at.roteskreuz.stopcorona.skeleton.core.screens.base.fragment.BaseFragment
+import kotlinx.android.synthetic.main.saved_ids_fragment.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+/**
+ * Screen to inform the user about how he can delete his exposure keys.
+ */
+class SavedIDsFragment : BaseFragment(R.layout.saved_ids_fragment) {
```
Comment (falkorichter) [inline app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt]: I find the name confusing, This is not about IDs, it´s about Temporarily Exposure Keys

Your KDoc explains it well.

Suggestion: **InfoOnDeleteTemporarilyExposureKeysFragment**

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    override fun onInitActionBar(actionBar: ActionBar?, toolbar: Toolbar?) {
+        super.onInitActionBar(actionBar, toolbar)
+        actionBar?.setDisplayHomeAsUpEnabled(true)
+        toolbar?.setNavigationIcon(R.drawable.ic_back)
+    }
+
+    override fun getTitle(): String? {
+        return getString(R.string.saved_IDs_title)
```
Comment (falkorichter) [inline app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt]: name of the Strings as well

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt (hunk 1 above)
Comment (mihbt) [reply app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt]: updated

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt (hunk 2 above)
Comment (mihbt) [reply app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt]: updated

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/InfoDeleteExposureKeysFragment.kt [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    override fun getTitle(): String? {
+        return getString(R.string.info_delete_exposure_keys_title)
+    }
+
+    private fun displayExposureNotificationsSettings() {
+        val notificationsSettingsIntent = viewModel.getExposureNotificationsSettingsIntent()
+        if (notificationsSettingsIntent.resolveActivity(requireContext().packageManager) != null) {
+            startActivity(notificationsSettingsIntent)
+        } else {
+            Toast.makeText(
```
Comment (dusanjencik) [inline app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/InfoDeleteExposureKeysFragment.kt:50]: Please also log an error. It's our mistake that we displayed this screen even the framework is not on the phone.

**Review by dusanjencik — DISMISSED** [review]
Nice 👍

File: app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/InfoDeleteExposureKeysFragment.kt (hunk 3 above)
Comment (falkorichter) [reply app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/InfoDeleteExposureKeysFragment.kt:50]: +1

Comment (mihbt) [reply app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/InfoDeleteExposureKeysFragment.kt:50]: updated

**Review by falkorichter — CHANGES_REQUESTED** [review]
just the silent error please!

**Review by falkorichter — APPROVED** [review]

**Pull request merged**